spec/openhab/log_spec.rb

```diff
@@ -57,16 +57,18 @@ def logger_name
 
     it "uses the rule id inside a timer block inside a rule" do
       executed = false
+      logger_name = nil
       rule "log test" do
         on_load
         run do
           after(1.second) do
             executed = true
-            expect(logger.name).to match(/^org\.openhab\.automation\.jrubyscripting\.rule\.log_spec\.rb:(?:\d+)$/)
```

Comment:
When this expectation isn't met, an exception(?) is raised inside the timer. This somehow caused the whole rspec not to complete. It would time out after about 7 minutes. So I've moved it outside the timer.

Reply:
well that sounds like something interesting to investigate, to prevent possible hanging builds in the future ;)